src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -1808,6 +1808,25 @@ namespace wallet_rpc
     };
   };
 
+  struct COMMAND_RPC_CHANGE_WALLET_PASSWORD
+  {
+    struct request
+    {
+      std::string old_password;
+      std::string new_password;
+
+      BEGIN_KV_SERIALIZE_MAP()
+        KV_SERIALIZE(old_password)
+        KV_SERIALIZE(new_password)
+      END_KV_SERIALIZE_MAP()
+    };
+    struct response
+    {
+      BEGIN_KV_SERIALIZE_MAP()
+      END_KV_SERIALIZE_MAP()
+    };
+  };
+
```

Comment:
Bump minor RPC version near thetop of the file.

Reply:
done